tardis/model/tests/data/abund.dat

```diff
@@ -1,4 +1,4 @@
-H He Ni56
+Index H He Ni56
```

Comment:
Discuss file name.

Reply:
Check other abund.dat files for consistency.

Reply:
https://tardis-sn.github.io/tardis/models/examples/abundancecust.html indicates that the cell index should always be present.  There are multiple abund.dat files where this is not the case.